app/queries/get_linkables.rb

```diff
@@ -15,30 +15,31 @@ def call
         .last
 
       Rails.cache.fetch ["linkables", document_type, latest_updated_at] do
-        edition_ids = Queries::GetEditionIdsWithFallbacks.(
-          Edition.with_document.distinct.where(
-            document_type: [document_type, "placeholder_#{document_type}"]
-          ).pluck('documents.content_id'),
-          state_fallback_order: [:published, :draft]
-        )
-
-        Edition.with_document
-          .where(id: edition_ids)
-          .pluck(:title, :content_id, :state, :base_path, "COALESCE(details->>'internal_name', title)")
-          .map do |(title, content_id, state, base_path, internal_name)|
-            LinkablePresenter.new(
-              title,
-              content_id,
-              state,
-              base_path,
-              internal_name,
-            )
-          end
+        linkables_query(document_type)
+          .map { |result| LinkablePresenter.new(*result) }
       end
     end
 
   private
 
     attr_reader :document_type
+
+    def linkables_query(document_type)
+      Edition.with_document
+        .where(
+          document_type: [document_type, "placeholder_#{document_type}"],
+          state: %w(published draft),
```

Comment:
Is there a difference in functionality between this and state_fallback_order: [:published, :draft] above?